unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/NFTs/BuilderInWorldNFTController.cs

```diff
@@ -0,0 +1,146 @@
+using DCL.Configuration;
+using DCL.Helpers.NFT;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Text;
+using UnityEngine;
+
+public class BuilderInWorldNFTController 
+{
+    public event System.Action OnNFTUsageChange;
+    NFTOwner nFTOwner;
+
+    Coroutine fechNFTsCor;
```

Comment:
```suggestion
    Coroutine fetchNFTsCoroutine;
```

Please avoid abbreviations unless absolutely necessary.